README.md

```diff
@@ -62,17 +72,48 @@ Username: postgres
 Password: (not necessary)
 ```
 
-6) Ensure the database `dbautoscheduler` has been created:
+7) Ensure the database `dbautoscheduler` has been created:
 Using psql: Type `\l` in the psql prompt and see if `dbautoscheduler` is in the list.
 Using pgAdmin: Click the name of the server you created, and see if `dbautoscheduler` is in the tree menu.
-7) If you'd like to get the current project running, you can clone the repository with `git clone https://github.com/gannonprudhomme/AutoScheduler/` and run the steps below.
+
+8) Navigate to the frontend directory: `./autoscheduler/frontend/src/` and run `npm install`
+    - This is for TypeScript compilation and testing.
+    
+9) Before making code changes, please go to the github project [wiki](https://github.com/aggie-coding-club/Rev-Registration/wiki) and follow the Project Setup and Install section.
 
 ## Running
 
 Before running any commands, if you're not running in the virtual environment(you should see `(env)` somewhere in your current terminal line), run `source env/bin/activate` or `./env/Scripts/activate`.
 
-You will have to make and apply migrations before running the server (and whenever our models are changed). To generate the files that django uses to apply migrations, run `./manage.py makemigrations`, and to then apply these to the database, run `./manage.py migrate`.
+**Server:**
+- You will have to make and apply migrations before running the server (and whenever our models are changed). To generate the files that django uses to apply migrations, run `./manage.py makemigrations`, and to then apply these to the database, run `./manage.py migrate`.
+    - `manage.py` can be found in the `./autoscheduler/` directory.
+
+- After migrating, you can run the server with `./manage.py runserver`. When running for the first time, you will have to configure `autoscheduler/autoscheduler/config/postgres-info.json` with your postgres username and password. This file will be automatically created if you run `./manage.py runserver`.
+
+- If `./manage.py` isn't working, try to use `python manage.py ...` or `python3 manage.py ...` or `py manage.py ...` in place of it.
+    - See python aliases above
+
+- The first time you run the server, you will need to scrape the course info from TAMU
+    - `python manage.py scrape_depts`
+    - `python manage.py scrape_courses`
+    - `python manage.py scrape_grades`
+        - You can put a term afterwards if you just want to scrape a specific term for quick testing:
+            - `python manage.py scrape_courses 202011`
+
```

Comment:
```suggestion
#### Backend

Scraping command arguments:
- `python manage.py scrape_depts`.
  - `-t [term]` or `--term [term]`
    - e.g. `scrape_depts -t 202031`
- `python manage.py scrape_courses`
  - `-t [term]` or `--term [term]`
    - e.g. `scrape_courses -t 202031`
  - `-y [year]` or `--year [year]`
    - e.g. `scrape_courses -y 2020`
    - Scrapes all semesters + locations for the given year
  - `-r` or `--recent`
    - e.g. `scrape_courses -r`
      - Scrapes the most recent terms. E.g. if it's October 1st, will scrape Summer + Fall 2020 for all locations
  - `-t`, `-y`, and `-r` are mutually exclusive
- `python manage.py scrape_grades`
  - `-y [year]` or `--year [year]`
    - e.g. `scrape_grades -y 2020`
  - `-c [college]` or `--college [college]`
    - e.g. `scrape_grades -c EN`
  - `-p [procs]` or `--procs [procs]`
    - e.g. `scrape_grades -p 5`
    - `scrape_grades` uses mutli-processing, and this changes how many processes run at a time. Defaults to the number of cores your CPU has.
  - Note that all of `scrape_grades` arguments can be used at the same time
```

Per the comment on the line above this about removing the `python manage.py scrape_courses 202011` documentation, made me realize that we should probably document it, so figured we should add this

Reply:
Actually on second thought this might not be the best place for it? Feel free to move it wherever you think it makes the most sense